scopes/workspace/install/install.main.runtime.ts

```diff
@@ -362,7 +365,7 @@ export class InstallMain {
       linkingOptions: options,
     });
     const compIds = await this.workspace.listIds();
-    const res = await linker.linkCoreAspectsAndLegacy(this.workspace.path, compIds, options);
+    const res = await linker.linkCoreAspectsAndLegacy(this.workspace.path, compIds, options.rootPolicy, options);
```

Comment:
why. is it added through new argument if rootPolicy is in the options?

Reply:
because in the `linkCoreAspectsAndLegacy` it's not part of the options (and I think it should be part of the options there)